wren-tests/src/test/java/io/wren/testing/TestConfigResource.java

```diff
@@ -46,6 +46,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+@Test(singleThreaded = true, description = "Every test will effect each other, so we need to run them in single thread.")
```

Comment:
IMO, `description` is used to explain why needs this test. I prefer to explain why use `singleThreaded` by comments.
```suggestion
//  description = "Every test will effect each other, so we need to run them in single thread."
@Test(singleThreaded = true)
```